helm/fcp-fd-comms/templates/config-map.yaml

```diff
@@ -3,10 +3,9 @@
 data:
   MESSAGE_QUEUE_HOST: {{ .Values.container.messageQueueHost | quote }}
   {{- if and (.Values.environmentCode) (not .Values.pr) }}
-  COMMS_SUBSCRIPTION_ADDRESS: {{ .Values.container.messagesSubscriptionAddress | quote }}-{{ .Values.environmentCode }}
-  COMMS_TOPIC_ADDRESS: {{ .Values.container.messagesTopicAddress | quote }}-{{ .Values.environmentCode }}
+  COMMS_SUBSCRIPTION_ADDRESS: {{ .Values.container.messagesSubscriptionAddress }}-{{ .Values.environmentCode }}
```

Comment:
Drop this out of the if statement entirely.  ie below the else too

Reply:
Done, this is what it looks like now:
```
{{- include "ffc-helm-library.container-config-map" (list . "fcp-fd-comms.config-map") -}}
{{- define "fcp-fd-comms.config-map" -}}
data:
  MESSAGE_QUEUE_HOST: {{ .Values.container.messageQueueHost | quote }}
  {{- if and (.Values.environmentCode) (not .Values.pr) }}
  COMMS_TOPIC_ADDRESS: {{ .Values.container.messagesTopicAddress }}-{{ .Values.environmentCode }}
  {{ else }}
  COMMS_TOPIC_ADDRESS: {{ .Values.container.messagesTopicAddress | quote }}
  {{- end }}
  NODE_ENV: {{ .Values.environment | quote }}
  PORT: {{ .Values.container.port | quote }}
  APPINSIGHTS_CLOUDROLE: {{ .Chart.Name | quote }}
{{- end -}}
```
I.e. just the topic is referred to in the `if` statement.